pom.xml

```diff
@@ -162,12 +162,6 @@
         <awaitility.version>4.1.1</awaitility.version>
         <hikari.version>4.0.3</hikari.version>
 
-        <sonar.jacoco.jar>${basedir}/lib/jacocoagent.jar</sonar.jacoco.jar>
```

Comment:
All there properties are just some leftovers or what's the reason for removing them?

Reply:
It was leftover from the configuration of sonar and jacoco.
Currently, only one property is required and configured in Jenkins:
`-Dsonar.coverage.jacoco.xmlReportPaths="${WORKSPACE}/hazelcast-enterprise-coverage-report/target/site/jacoco-aggregate/jacoco.xml" \`